website/pages/docs/calendar/configuration.mdx

```diff
@@ -39,6 +39,19 @@ const config = {
    * */
   defaultView: viewMonthGrid.name,
 
+   /**
+   * Toggle automatic view change when the calendar is resized below a certain width breakpoint.
+   * Defaults to true
+   * */
+  isResponsive: false,
+
+   /**
+   * Pass a function that will be used to calculate a break point used for calculating the current view type.
+   * Given standard browser settings where 1rem = 16px, the calendar has an default break point of 700px.
+   * More information on the default function: https://schedule-x.dev/docs/calendar/views#a-note-on-screen-sizes
+   * */
+  customBreakpointFunction: () => window.innerWidth <= 800,
```

Comment:
I think we can move this into the `callbacks` property of the configuration object, and add the `$app` object as a parameter, to supercharge the callback with a bit of extra possibilities. That way the implementer has access to all of the important internal APIs when calculating whether the calendar is small.

```suggestion
  isCalendarSmall: ($app: CalendarAppSingleton) => window.innerWidth <= 800,
```

Reply:
Agreed